.github/workflows/release.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+        with:
```

Comment:
We need to specify the branch during checkout because this script is triggered from a tag publish, and we need to checkout to the correct branch because the last step is to commit the bumped files